enforcer/src/main/java/org/wso2/micro/gateway/enforcer/security/jwt/JWTUtil.java

```diff
@@ -131,5 +158,117 @@ public static boolean verifyTokenSignature(SignedJWT jwt, String alias) throws M
         }
     }
 
+    public static Certificate getCertificate() {
```

Comment:
Introduced a separate package called TLSUtils
Shall we move this code to that package.